proto/expression.proto

```diff
@@ -11,6 +11,41 @@ option (gogoproto.marshaler_all) = true;
 option (gogoproto.sizer_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
+// MySQL data types.
+enum DataType {
+	TypeDecimal = 0;
+	TypeTiny = 1;
+	TypeShort = 2;
+	TypeLong = 3;
+	TypeFloat = 4;
+	TypeDouble = 5;
+	TypeNull = 6;
+	TypeTimestamp = 7;
+	TypeLongLong = 8;
+	TypeInt24 = 9;
+
+	TypeDuration = 11;
+	TypeDatetime = 12;
+	TypeYear = 13;
+
+	TypeVarchar = 15;
+	TypeBit = 16;
+
+	TypeJSON = 0xf5;
+	TypeNewDecimal = 0xf6;
+
+	TypeTinyBlob = 0xf9;
+	TypeMediumBlob = 0xfa;
+	TypeLongBlob = 0xfb;
+	TypeBlob = 0xfc;
+	TypeVarString = 0xfd;
+	TypeString = 0xfe;
+}
+
+message FieldType {
```

Comment:
Looks like these fields are needed, we could add them later when the code is written where these fields are used.